tests/build_/test_arrangements.py

```diff
@@ -1703,3 +1729,319 @@ def test_export_operator_manifests(self, osbs, extract_platform):
         plugins = get_plugins_from_build_json(build_json)
         args = plugin_value_get(plugins, 'postbuild_plugins', 'export_operator_manifests', 'args')
         assert match_args == args
+
+    @pytest.mark.parametrize('scratch', [False, True])  # noqa:F811
+    @pytest.mark.parametrize('base_image', ['koji/image-build', 'foo'])
+    def test_koji_parent_in_orchestrator(self, osbs, base_image, scratch):
+        additional_params = {
+            'base_image': base_image,
+        }
+        if scratch:
+            additional_params['scratch'] = True
+        _, build_json = self.get_orchestrator_build_request(osbs, additional_params)
+        plugins = get_plugins_from_build_json(build_json)
+
+        if scratch:
+            with pytest.raises(NoSuchPluginException):
+                get_plugin(plugins, 'prebuild_plugins', PLUGIN_KOJI_PARENT_KEY)
+        else:
+            get_plugin(plugins, 'prebuild_plugins', PLUGIN_KOJI_PARENT_KEY)
+            with pytest.raises(KeyError):
+                plugin_value_get(plugins, 'prebuild_plugins', PLUGIN_KOJI_PARENT_KEY, 'args')
+
+    @pytest.mark.parametrize('scratch', [False, True])  # noqa:F811
+    def test_koji_upload(self, scratch, osbs):
+        additional_params = {
+            'base_image': 'fedora:latest',
+            'koji_upload_dir': 'upload',
+        }
+        if scratch:
+            additional_params['scratch'] = True
+        _, build_json = self.get_worker_build_request(osbs, additional_params)
+        plugins = get_plugins_from_build_json(build_json)
+
+        args = plugin_value_get(plugins, 'postbuild_plugins', PLUGIN_KOJI_UPLOAD_PLUGIN_KEY, 'args')
+        expected_args = {
+            'blocksize': 10485760,
+            'koji_upload_dir': 'upload',
+            'platform': 'x86_64',
+            'report_multiple_digests': True
+        }
+        assert args == expected_args
+
+    def test_koji_import(self, osbs):  # noqa:F811
+        additional_params = {
+            'base_image': 'fedora:latest',
+            'koji_upload_dir': 'upload',
+        }
+        params, build_json = self.get_orchestrator_build_request(osbs, additional_params)
+        plugins = get_plugins_from_build_json(build_json)
+
+        with pytest.raises(NoSuchPluginException):
+            get_plugin(plugins, 'postbuild_plugins', PLUGIN_KOJI_UPLOAD_PLUGIN_KEY)
```

Comment:
Does this actually test koji_import? Maybe we could at least add something like this:
```python
get_plugin(plugins, 'exit_plugins', PLUGIN_KOJI_IMPORT_PLUGIN_KEY)
with pytest.raises(KeyError):
    plugin_value_get(plugins, 'exit_plugins', PLUGIN_KOJI_IMPORT_PLUGIN_KEY, 'args')
```

Reply:
Fixed.